src/metrics/mongoose_metrics_mam_hooks.erl

```diff
@@ -34,59 +34,60 @@
 %%-------------------
 
 %% @doc Here will be declared which hooks should be registered when mod_mam_pm is enabled.
--spec get_mam_hooks(_) -> gen_hook:hook_list().
-get_mam_hooks(HostType) ->
+-spec get_mam_hooks(mongooseim:host_type()) -> gen_hook:hook_list().
+get_mam_hooks(Host) ->
```

Comment:
All these varnames should stay as `HostType`s, here and in the function below as well.